homeassistant/components/keba/__init__.py

```diff
@@ -180,6 +180,11 @@ def add_update_listener(self, listener):
         # initial data is already loaded, thus update the component
         listener()
 
+    async def async_request_data(self, param):
+        """Request new data in async way."""
+        await self.request_data()
+        _LOGGER.debug("New data from KEBA wallbox requested.")
```

Comment:
We don't end logging messages with period.

Reply:
Sorry, forgot about that.